src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.controller;
+
+import javax.validation.Valid;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/authentication")
```

Comment:
as for me, you can remove this. In task we should use ../register only

Reply:
agree, i dont think we need "/authentication" mapping here